autometa/common/external/samtools.py

```diff
@@ -37,32 +56,42 @@ def run(cmd):
     logger.debug(f'cmd: {cmd}')
     with open(os.devnull, 'w') as stdout, open(os.devnull, 'w') as stderr:
         retcode = subprocess.call(cmd, stdout=stdout, stderr=stderr, shell=True)
-    if retcode:
+    if not retcode:
+        return True
+    else:
         logger.warning(f'args:{cmd} retcode:{retcode}')
         return False
-    return True
 
-def sort(sam, out, nproc=0):
-    cmd = f'samtools view -@{nproc} -bS {sam} | samtools sort -@{nproc} -o {out}'
-    run(cmd)
+def sort(sam, bam, nproc=mp.cpu_count()):
+    """
+    Views the sam file and then sorts the alighnments by leftmost coordinates.
+    Ouputs the sorted alighnments in the specified bam file.
```

Comment:
You could probably place this in the previous sentence.

I.e. Views then sorts `sam` by leftmost coordinates and outputs to `bam`